### PR TITLE
[secrets][m]: script to update secrets automatically - fixe #59

### PR DESCRIPTION
**Pull request by zelima**
Script to update secrets on kubernetes with one simple command and `.env` file

### CODE REVIEW

File: update_secrets.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                    env_value = configfile[service]['environment'][env_]
+                    to_replace = pattern.findall(str(env_value))
+                    for rpl in to_replace:
+                        configfile[service]['environment'][env_] = env.get(rpl[2:-1], '')
+        return configfile
+
+
+    def _run_commands(self, cmd, options=''):
+        out = ''
+        cmd = cmd.split(' ') + [options]
```
Comment (akariv) [inline update_secrets.py]: just use split():
```
>>> "sfdsfds     fsdfsdds   fds".split()
['sfdsfds', 'fsdfsdds', 'fds']
>>> "sfdsfds     fsdfsdds   fds".split(" ")
['sfdsfds', '', '', '', '', 'fsdfsdds', '', '', 'fds']
```

File: update_secrets.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            self._run_commands(cmd_del % (serv, stage))
+            print('Creating new secrets for %s' % serv)
+            self._run_commands(cmd, options)
+            cmd = 'kubectl create secret generic %s-%s '
+
+
+# ==============================================
+# CLI
+
+def _object_methods(obj):
```
Comment (akariv) [inline update_secrets.py]: why do we need this just to run one method?

File: update_secrets.py (hunk 1 above)
Comment (akariv) [reply update_secrets.py]: And [`shlex.split`](https://docs.python.org/3/library/shlex.html#shlex.split) is even better.

File: update_secrets.py (hunk 2 above)
Comment (zelima) [reply update_secrets.py]: Yes for now, but usually, we add more methods to the later

**Pull request merged**